app/src/main/java/webdevils/webdevilsapp/LoginActivity.java

```diff
@@ -1,19 +1,29 @@
 package webdevils.webdevilsapp;
 
 import android.content.Intent;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
+import java.util.HashMap;
+import android.content.DialogInterface;
+import android.app.AlertDialog.Builder;
+
 
 public class LoginActivity extends AppCompatActivity {
 
+    HashMap<String, String> users = new HashMap<String, String>();
```

Comment:
For now we can change...
HashMap<String, String> users = new HashMap<String, String>(); 
to two strings...
String testUserName = "user";
String testPassword = "password";
(you'll have to update your validateUser() method ...) 
or we can change ...
HashMap<String, String> users
to
HashMap<String, String> testUser